sys/include/suit/v3/handlers.h

```diff
@@ -0,0 +1,133 @@
+/*
+ * Copyright (C) 2019 Koen Zandberg
+ *               2019 Kaspar Schleiser <kaspar@schleiser.de>
+ *               2019 Inria
+ *               2019 Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+/**
+ * @ingroup     sys_suit_v3
+ * @brief       SUIT v3 manifest handlers
+ *
+ * @experimental
+ *
+ * @{
+ *
+ * @brief       Handler functions for SUIT manifests
+ * @author      Koen Zandberg <koen@bergzand.net>
+ * @author      Kaspar Schleiser <kaspar@schleiser.de>
+ */
+
+#ifndef SUIT_V3_HANDLERS_H
+#define SUIT_V3_HANDLERS_H
+
+#include <stddef.h>
+#include <stdint.h>
+
+#include "suit/v3/suit.h"
+#include "uuid.h"
+#include "nanocbor/nanocbor.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @name SUIT container identifiers
+ * @{
+ */
+#define SUIT_CONTAINER_VERSION              (1)
+#define SUIT_CONTAINER_SEQ_NO               (2)
+#define SUIT_CONTAINER_COMMON               (3)
+#define SUIT_CONTAINER_DEPS_RESOLUTION      (7)
+#define SUIT_CONTAINER_PAYLOAD_FETCH        (8)
+#define SUIT_CONTAINER_INSTALL              (9)
+#define SUIT_CONTAINER_VALIDATE            (10)
+#define SUIT_CONTAINER_LOAD                (11)
+#define SUIT_CONTAINER_RUN                 (12)
+#define SUIT_CONTAINER_TEXT                (13)
+/* @} */
+
+/**
+ * @name SUIT condition identifiers
+ * @{
+ */
+#define SUIT_COND_VENDOR_ID     (1)
+#define SUIT_COND_CLASS_ID      (2)
+#define SUIT_COND_IMAGE_MATCH   (3)
+#define SUIT_COND_USE_BEFORE    (4)
+#define SUIT_COND_COMPONENT_OFFSET      (5)
+#define SUIT_COND_DEVICE_ID            (24)
+#define SUIT_COND_IMAGE_NOT_MATCH      (25)
+#define SUIT_COND_MIN_BATTERY          (26)
+#define SUIT_COND_UPDATE_AUTHZ         (27)
+#define SUIT_COND_VERSION              (28)
+/* @} */
+
+/**
+ * @name SUIT directive identifiers
+ * @{
+ */
+#define SUIT_DIR_SET_COMPONENT_IDX  (12)
+#define SUIT_DIR_SET_DEPENDENCY_IDX (13)
+#define SUIT_DIR_ABORT              (14)
+#define SUIT_DIR_TRY_EACH           (15)
+#define SUIT_DIR_PROCESS_DEPS       (18)
+#define SUIT_DIR_SET_PARAM          (19)
+#define SUIT_DIR_OVERRIDE_PARAM     (20)
+#define SUIT_DIR_FETCH              (21)
+#define SUIT_DIR_COPY               (22)
+#define SUIT_DIR_RUN                (23)
+#define SUIT_DIR_WAIT               (29)
+#define SUIT_DIR_RUN_SEQUENCE       (30)
+#define SUIT_DIR_RUN_WITH_ARGS      (31)
+#define SUIT_DIR_SWAP               (32)
+/** @} */
+
+/**
+ * @brief suit handler prototype
+ *
+ * @param manifest  SUIT v3 manifest context
```

Comment:
Here too, how about being precise in the comments/doc and specify draft-ietf-suit-manifest-03 instead of just "v3" ?

Reply:
Fixed